Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -139,6 +139,19 @@ class SentryFileManagerTests: XCTestCase {
         let actualData = envelopes[0].contents
         XCTAssertEqual(expectedData, actualData as Data)
     }
+
+    func testDeleteOldEnvelopes() throws {
```

Comment:
`m`: Please add a test case where the folder is older than 90 days to ensure we only delete files in that folder older than 90 days and not the folder. If that happened, it would be nasty.

Reply:
Done